### ISSUE TITLE
cannot find package "grumpy/lib/io"

### ISSUE THREAD

**Issue by bigcmos** [google/grumpy#81]
I am using the python io libraries to read and write files in py python script and I'm getting an erorr trying to import io.  Below is a simplified version that demonstrates the error.  Is the python io library supported?

$ echo "import io" | make run
/tmp/tmpdlxl4K.go:4:2: cannot find package "grumpy/lib/io" in any of:
	/usr/local/go/src/grumpy/lib/io (from $GOROOT)
	/opt/home/daden/dev/grumpy/build/src/grumpy/lib/io (from $GOPATH)
Makefile:78: recipe for target 'run' failed
make: *** [run] Error 1

**Comment by Ayenko1980** [comment]
U run wake up to the reality,,,call my number you will face the consequenze

**Comment by Ayenko1980** [comment]
I am agent from washington

**Comment by ns-cweber** [comment]
No, `io` is not yet supported: https://github.com/google/grumpy/wiki/Missing-Features#standard-libraries

**Comment by JTAG1979** [comment]
Omgoodness gracious! I am pooping in my pants. Hahahahaha!

**Comment by bigcmos** [comment]
Is there a plan to support this library any time soon?

**Comment by trotterdylan** [comment]
The PyPy version doesn't seem to have too many dependencies: https://bitbucket.org/pypy/pypy/src/17b03637ebc2fd9269f12a8aabb614440eabf7a9/lib-python/2.7/_pyio.py

It might not be that hard to port.
